recipe/meta.yaml

```
@@ -7,29 +7,32 @@ package:
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
 
 build:
   number: 1
-  script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
+    - wheel
+    - setuptools
   run:
-    - python >=3.6
-    - numpy
-    - scipy >=0.14.0
-    - scikit-learn >=0.19.1
-    - matplotlib-base
-    - pyaml
-    - joblib
+    - python
+    - {{ pin_compatible('numpy') }}
+    - scipy >=0.19.0
```

```suggestion
    - numpy >=1.13.3
    - scipy >=0.19.1
```
https://github.com/scikit-optimize/scikit-optimize/blob/a2369ddbc332d16d8ff173b12404b03fea472492/setup.py#L45C62-L46

---

thanks!